drivers/dai/intel/dmic/dmic.c

```diff
@@ -607,18 +607,16 @@ static void dai_dmic_start(struct dai_intel_dmic *dmic)
 		}
 	}
 
-#ifndef CONFIG_SOC_SERIES_INTEL_ACE
 	/* Clear soft reset for all/used PDM controllers. This should
 	 * start capture in sync.
 	 */
 	for (i = 0; i < CONFIG_DAI_DMIC_HW_CONTROLLERS; i++) {
 		dai_dmic_update_bits(dmic, base[i] + CIC_CONTROL,
-				     CIC_CONTROL_SOFT_RESET_BIT, 0);
+				     CIC_CONTROL_SOFT_RESET_BIT|CIC_CONTROL_MIC_MUTE_BIT, 0);
```

Comment:
I wonder if it would be better to have two separate commits, one to apply this code on ACE as well, and second commit to add the MUTE bit. 

Minor style note: empty space around bit operations is preferred, i.e. "CIC_CONTROL_SOFT_RESET_BIT | CIC_CONTROL_MIC_MUTE_BIT"

Reply:
I'm still working on it. I have some cases are failing today, trying to make sure everything is covered by this patch